src/nvme/util.h

```diff
@@ -148,12 +148,14 @@ void nvme_init_copy_range_f1(struct nvme_copy_range_f1 *copy, __u16 *nlbs,
  * @fid:   Feature identifier, see &enum nvme_features_id.
  * @cdw11: The cdw11 value may affect the transfer (only known fid is
  *	   %NVME_FEAT_FID_HOST_ID)
+ * @dir:   Data transfer direction: false - host to controller, true -
+ *	   controller to host may affect the transfer (only known fid is
+ *	   %NVME_FEAT_FID_HOST_MEM_BUF).
  * @len:   On success, set to this features payload length in bytes.
- *
  * Return: 0 on success, -1 with errno set to EINVAL if the function did not
  * recognize &fid.
  */
-int nvme_get_feature_length(int fid, __u32 cdw11, __u32 *len);
+int nvme_get_feature_length(int fid, __u32 cdw11, bool dir, __u32 *len);
```

Comment:
Thanks for your comments. About the suggestion mensioned to make sure let me confirm below.
If added nvme_get_feature_length2() but still the original API does not work correctly for the feature ID reported by the issue. Is this really okay?

Reply:
Yes, we need keep the `nvme_get_feature_length()` around, because removing or changing this function is not possible as we promised not to break the API.

Reply:
I see. Thanks for the explanation.

Reply:
Just pushed the changes as suggested.